package/kernel/linux/modules/netdevices.mk

```diff
@@ -523,6 +523,7 @@ endef
 
 $(eval $(call KernelPackage,igb))
 
+
```

Comment:
Unnecessary whitespace.

Reply:
Whitespace followed the conventions already in the file - two empty lines before each "define" block

Reply:
That may be, but each commit also needs to be singular in purpose and focus.  We don't hit "targets of opportunity" because that makes backing out a bad commit too problematic.

In general, use the "light touch" when working on the code base and you can't go wrong.